src/cryptoadvance/specter/util/bitcoind_setup_tasks.py

```diff
@@ -11,6 +11,11 @@
 
 def setup_bitcoind_thread(specter=None, internal_bitcoind_version=""):
     try:
+        # There is no bitcoind binary for Raspberry Pi 4 (armv7l)
+        if platform.system() == "Linux" and "armv" in platform.machine():
+            raise Exception(
```

Comment:
I think for Bitcoin Core actually https://bitcoincore.org/bin/bitcoin-core-0.21.0/bitcoin-0.21.0-arm-linux-gnueabihf.tar.gz should work on Pi, so maybe we can allow that?

Reply:
Confirmed, that binary works on Raspberry Pi OS. But running into a new issue. Will note that below.